teamBangazon/BangazonAPI
# added OrderController.cs and Models/Orders.cs

### Changes Made:
Added OrderController.cs and Models/Orders.cs

### Steps to Test:
On LOCAL MACHINE, inside the TERMINAL: 
git fetch --all
 git checkout es_order

INSIDE VISUAL CODE: 
delete MIGRATIONS and DATABASE

In VISUAL CODE TERMINAL: 
dotnet ef migrations add initial 
dotnet ef database update
 dotnet run

INSIDE POSTMAN:
place   http://localhost:5000/api/order/   into the url

POST: select BODY
use {
         "Customer": "Bob",
         "PaymentType": "Visa",
         "CreatedOn": "2017-07-27"
       }

GET: inside the URL add and ID to the end to select that order or add nothing to get all available orders.

PUT: select BODY
Make sure the appropriate id is added to the end of the URL that corresponds to the order you want to edit. Make sure "OrderId": 2, is added above "Customer". "2" is the same id that you put into the URL.

DELETE: place the correct ID at the end of the URL that you want to delete.

### Ref URL
[4](https://github.com/teamBangazon/BangazonAPI/issues/4)
